res/css/views/audio_messages/_PlaybackContainer.scss

```diff
@@ -96,13 +85,17 @@ limitations under the License.
                 background-color: transparent;
             }
         }
+    }
+}
 
-        // For timeline-rendered playback, the clock is on the other side of the waveform.
-        & + .mx_Clock {
-            text-align: right;
```

Comment:
This text-align was fixing a layout issue that is introduced by this PR where the clock ends up not matching the designs:
![image](https://user-images.githubusercontent.com/1190097/173901937-28b0501c-0421-477b-bf21-500379336168.png)

(this PR)

There should be no padding or margin on the right side of the clock.